portal-bridge/src/beacon_bridge.rs

```diff
@@ -292,6 +292,17 @@ impl BeaconBridge {
         let data = api.get_lc_updates(expected_current_period, 1).await?;
         let update: Value = serde_json::from_str(&data)?;
         let update: LightClientUpdateCapella = serde_json::from_value(update[0]["data"].clone())?;
+
+        // We don't serve a `LightClientUpdate` if its finalized header slot is not within the expected current period.
+        if update.finalized_header.beacon.slot / (32 * 256) != expected_current_period {
```

Comment:
*nitpick* I'd prefer to see `(32 * 256)` to be a const w/ a descriptive name. Seems like we can also set this value (`update.finalized_header.beacon.slot / (32 * 256)`) to a variable (eg `actual_current_period`) and then re-use it in the `warn!` log rather than recalculating it.